src/integrationTest/kotlin/com/sourcegraph/cody/chat/ChatTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.sourcegraph.cody.chat
+
+import com.intellij.testFramework.runInEdtAndGet
+import com.sourcegraph.cody.util.CodyIntegrationTextFixture
+import com.sourcegraph.cody.util.CustomJunitClassRunner
+import com.sourcegraph.cody.util.TestingCredentials
+import java.util.concurrent.TimeUnit
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.until
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(CustomJunitClassRunner::class)
+class ChatTest : CodyIntegrationTextFixture() {
+  override fun myCredentials() = TestingCredentials.enterprise
+
+  @Test
+  fun testRemoteContextFileItems() {
+    val session = runInEdtAndGet {
+      val s = AgentChatSession.createNew(project)
+      s.sendMessage("what is json rpc", emptyList())
+      s
+    }
+
+    await.atMost(10, TimeUnit.SECONDS).ignoreExceptions() until {
+      val messages = session.messages
+      session.messages.size == 2 && !messages[1].contextFiles.isNullOrEmpty()
```

Comment:
Shouldn't we also check the exact paths?

Reply:
yes